android/brave_java_resources.gni

```diff
@@ -794,6 +794,7 @@ brave_java_resources = [
   "java/res/layout/brave_custom_tabs_toolbar.xml",
   "java/res/layout/brave_dialog_preference.xml",
   "java/res/layout/brave_exit_confirmation.xml",
+  "java/res/layout/brave_leo_clear_history_dialog.xml",
```

Comment:
@bridiver would it be okay if we landed this `brave_java_resources` addition? I'm not an Android expert, so it'll take me a while to work out the proper way of doing this and I'm going on PTO on Monday

Reply:
this is fine for now